### PR TITLE
Upgrade to Play Google auth 0.5.0

### PR DESCRIPTION
**Pull request by LATaylor-guardian**

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
@adamnfish Much nicer.

**Comment by mchv** [comment]
👍

File: app/com/gu/bard/auth/AuthActions.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import com.gu.googleauth._
+import play.api.Configuration
+import controllers.routes
+
+trait AuthActions extends Actions {
+  def conf: Configuration
+
+  override val authConfig = GoogleAuthConfig(
+    clientId = conf.getString("google.clientId").getOrElse(sys.error(s"Missing config key: google.clientId")),
```
Comment (adamnfish) [inline app/com/gu/bard/auth/AuthActions.scala]: You could put `GoogleAuthConfig` into `AppComponents` to keep all your config reading in one place, but fine like this as well - you'd have to wire something in either way so no huge difference.

Comment (LATaylor-guardian) [inline app/com/gu/bard/auth/AuthActions.scala]: I did have it in there but moved it out as I was blindly copying your example. But good point, I'll move it back out.

**Pull request merged**